src/mom5/ocean_csiro_bgc/csiro_bgc.F90

```diff
@@ -1003,10 +1004,12 @@ subroutine csiro_bgc_sbc(isc, iec, jsc, jec, isd, ied, jsd, jed, &
 
 do j = jsc, jec  !{
   do i = isc, iec  !{
-    sc_co2(i,j) = 2073.1 + t_prog(indtemp)%field(i,j,1,time%taum1) * &
-         (-125.62 + t_prog(indtemp)%field(i,j,1,time%taum1) *        &
-          (3.6276 + t_prog(indtemp)%field(i,j,1,time%taum1) *        &
-           (-0.043219))) * grid%tmask(i,j,1)
+    ! Limit temp used for Schmidt number calculation.
+    schmidt_temp = min(35.0, t_prog(indtemp)%field(i,j,1,time%taum1))
+    sc_co2(i,j) = 2073.1 + schmidt_temp * &
+        (-125.62 + schmidt_temp *         &
+        (3.6276 + schmidt_temp *          &
+        (-0.043219))) * grid%tmask(i,j,1)
```

Comment:
I don't expect this to be fixed, but felt duty bound to say "wow there are a lot of magic numbers here".

Reply:
Hocus pocus